src/index.ts

```diff
@@ -106,8 +106,6 @@ const getArguments = (args) => {
         } else if (arg.value.kind === "Variable") {
             argsObj[arg.name.value] =
                 `${arg.value.name.value}${isVariableDropinConst}`
-        } else if (arg.selectionSet) {
-            argsObj[arg.name.value] = getSelections(arg.selectionSet.selections)
```

Comment:
Additionally...

getSelections is used in two places:

  1. Line 134: Recursively called within itself to handle nested selection sets in
   fields:
  if (selection.selectionSet) {
      selObj[selectionName] = getSelections(selection.selectionSet.selections)
  }
  2. Line 222: Called from the main graphQlQueryToJson function to process the
  root selection set:
  const selections = getSelections(firstDefinition.selectionSet.selections)

  The function processes field selections (like user { name email }), not argument
   selections. Fields can have nested selection sets, but arguments cannot - which
   is why the arg.selectionSet branch was dead code.